tech.html

```diff
@@ -81,6 +81,428 @@ <h1>Autonomous Vehicles (AV)</h1>
             <p>Personally, having access to a fully autonomous car would be life-changing. The convenience of being driven through peak-hour traffic and only having to focus on finishing my book or study, would result in a stress-free journey. I will save time as there will be no need to worry about finding a car park. The option of being driven interstate to see my family, subject to available infrastructure, instead of getting on a plane, with the ability to sleep all the way there, would be brilliant. These types of cars would help my elderly family members and friends and eventually myself, to keep our independence in our aging years.</p>
             <p>Having said all that, I wouldn’t use an autonomous car until they had been cleared as 100% reliable and safe. With so much computerisation involved, they are susceptible to hacking and malfunction. If I had the choice between a Level 5, fully automated with no access to a steering wheel and a Level 4, most functions automated except under unusual circumstances, I would choose Level 4, as I don’t completely trust the way the world is heading in relation to the advancements in technology. I feel like it is about to supersede the need for humans in every sector. Sometimes it’s important to stop and look around at what we’ve got before it’s gone forever.</p>
               <p>ADD REFERENCES</p>
+              <img src="img/NLP.png" alt="NLP" width="460" height="345">
+
+<p>
+
+    Natural Language processing and chatterbots
+
+</p>
+
+<p>
+
+    <strong>What does it do?</strong>
```

Comment:
I don't think we need to include the question headings. I didn't.

Reply:
I'll delete them.